backend_new/test/integration/multiselect-question.e2e-spec.ts

```diff
@@ -0,0 +1,309 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { INestApplication } from '@nestjs/common';
+import * as request from 'supertest';
+import { AppModule } from '../../src/app.module';
+import { PrismaService } from '../../src/services/prisma.service';
+import { multiselectQuestionFactory } from '../../prisma/seed-helpers/multiselect-question-factory';
+import { jurisdictionFactory } from '../../prisma/seed-helpers/jurisdiction-factory';
+import { MultiselectQuestionCreate } from '../../src/dtos/multiselect-questions/multiselect-question-create.dto';
+import { MultiselectQuestionUpdate } from '../../src/dtos/multiselect-questions/multiselect-question-update.dto';
+import { IdDTO } from 'src/dtos/shared/id.dto';
+import { MultiselectQuestionsApplicationSectionEnum } from '@prisma/client';
+import { stringify } from 'qs';
+import { MultiselectQuestionQueryParams } from '../../src/dtos/multiselect-questions/multiselect-question-query-params.dto';
+import { Compare } from '../../src/dtos/shared/base-filter.dto';
+import { randomUUID } from 'crypto';
+
+describe('MultiselectQuestion Controller Tests', () => {
+  let app: INestApplication;
+  let prisma: PrismaService;
+  let jurisdictionId: string;
+
+  beforeAll(async () => {
+    const moduleFixture: TestingModule = await Test.createTestingModule({
+      imports: [AppModule],
+    }).compile();
+
+    app = moduleFixture.createNestApplication();
+    prisma = moduleFixture.get<PrismaService>(PrismaService);
+    await app.init();
+
+    const jurisdiction = await prisma.jurisdictions.create({
+      data: jurisdictionFactory(41),
+    });
+    jurisdictionId = jurisdiction.id;
+  });
+
+  it('testing list endpoint without params', async () => {
```

Comment:
@YazeedLoonat and @ludtkemorgan , didn't we decide that tests were going to clean up the data they create or does some benchmarking need to happen first?

Reply:
we did want to do some benchmarking before we made the switch

We wanted to know:
1. what a switch to runInBand will do
2. what does clearing the db between runs means for CI and the CI pipeline 

I didn't want to tack that in here, so pushing that off to a stand alone PR/testing process